include/GLFW/glfw3.h

```diff
@@ -3304,6 +3356,43 @@ GLFWAPI void glfwSetWindowTitle(GLFWwindow* window, const char* title);
  */
 GLFWAPI void glfwSetWindowIcon(GLFWwindow* window, int count, const GLFWimage* images);
 
+/*! @brief Sets the taskbar progress for the specified window.
+ *
+ *  This function sets the taskbar progress of the specified window.
+ *
+ *  @param[in] window The window whose taskbar progress to set.
+ *  @param[in] progressState State of the progress to be displayed in the taskbar. Valid values are:
+ *  @ref GLFW_TASKBAR_PROGRESS_NOPROGRESS, @ref GLFW_TASKBAR_PROGRESS_INDETERMINATE,
+ *  @ref GLFW_TASKBAR_PROGRESS_NORMAL, @ref GLFW_TASKBAR_PROGRESS_ERROR
+ *  and @ref GLFW_TASKBAR_PROGRESS_PAUSED.
+ *  @param[in] completed The amount of completed progress to set. Valid range is 0 to 100.
+ *  This is ignored if progressState is set to @ref GLFW_TASKBAR_PROGRESS_NOPROGRESS.
+ *
+ *  @errors Possible errors include @ref GLFW_NOT_INITIALIZED, @ref
+ *  GLFW_INVALID_VALUE, @ref GLFW_INVALID_ENUM, @ref GLFW_PLATFORM_ERROR,
+ *  @ref GLFW_FEATURE_UNIMPLEMENTED and @ref GLFW_FEATURE_UNAVAILABLE (see remarks).
+ *
+ *  @remark @win32 On Windows Vista and earlier, this function will emit @ref GLFW_FEATURE_UNAVAILABLE.
+ *
+ *  @remark @macos This function will emit @ref GLFW_FEATURE_UNIMPLEMENTED.
+ *
+ *  @remark @x11 @wayland Requires a valid application desktop file with the same name
+ *  as the compiled executable. Due to limitations in the Unity Launcher API
+ *  @ref GLFW_TASKBAR_PROGRESS_INDETERMINATE, @ref GLFW_TASKBAR_PROGRESS_ERROR and @ref GLFW_TASKBAR_PROGRESS_PAUSED
+ *  have the same behaviour as @ref GLFW_TASKBAR_PROGRESS_NORMAL. The Unity Launcher API is only known
+ *  to be supported on the Unity and KDE desktop environments, on other desktop environments this
+ *  function may do nothing.
+ *
+ *  @thread_safety This function must only be called from the main thread.
+ *
+ *  @sa @ref window_taskbar_progress
+ *
+ *  @since Added in version 3.4.
+ *
+ *  @ingroup window
+ */
+GLFWAPI void glfwSetWindowTaskbarProgress(GLFWwindow* window, const int progressState, int completed);
```

Comment:
The `completed` parameter seems like a boolean parameter. I think it's better described simply with the generic name "value".

Reply:
Done a480de9.